package.json

```
@@ -34,16 +45,18 @@
   },
   "homepage": "https://github.com/eslint/eslintrc#readme",
   "devDependencies": {
-    "chai": "^4.2.0",
-    "eslint": "^7.21.0",
+    "c8": "^7.7.3",
+    "chai": "^4.3.4",
+    "eslint": "^7.31.0",
     "eslint-config-eslint": "^7.0.0",
-    "eslint-plugin-jsdoc": "^32.2.0",
+    "eslint-plugin-jsdoc": "^35.4.1",
     "eslint-plugin-node": "^11.1.0",
     "eslint-release": "^3.1.2",
-    "fs-teardown": "0.1.1",
-    "mocha": "^8.1.1",
+    "fs-teardown": "0.1.3",
```

```suggestion
    "fs-teardown": "^0.1.3",
```

This was pinned in https://github.com/eslint/eslintrc/pull/45 to avoid `0.1.3` (doesn't work on Node 10.12.0). Now we can use `^` again.